tests/remote_provider.json

```diff
@@ -1,8 +1,8 @@
 {
   "host_bucket": "go.sdk.test",
-  "host": "172.28.41.39",
-  "access_key": "abcdabcd",
-  "secret_key": "abcdabcd",
-  "port": 9000
+  "host": "192.168.13.1",
```

Comment:
This isn't really any better than the previous hardcoded values but at least it aligns with the way a remote provider can be set up with our Vagrant Datera VM